config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+O4gOaR86U57yhCYcspGjvCz43aUeZ3W5gdpYOPJgpb+LHdF/iadTW3/galVeXxXysVmcRhyyhlB/BimtpZFvTyJxGfkNjL57EXAHoD7XAx7eTSdxm5c3dWIQazSncDb6LDVSeTDlMmZuu7gIONVLpzG9AqvuGEQ5xfhZnFdFCgQbEodgv87MVbfM7CMS3Ojg5Q5Ofge1I6JRwr24iCH8zHkUflL359uU5ZuSuGFlWHuAjOabEvZMXVeeWcFUrn0vMwnnY1PMvf+m/a4xdrHkFKrY1XIeMpmOEJmUE5oc6bth+Eg/XEes0/Mf8jxjlGRT1Emiq2LwAZozDDMAWwrwU9cPBn/LPSTwphiCN6z4/dn5tTTUw98T9LJ+QOnueQMyoZh4s4MgkFQpl99B/F2ev7bi3XLPMq93ew4C--hvqE2QHIgueuJVVh--TdJuM9VtVhdS+LIhdN1KdQ==
```

Comment:
I am not up on the 5.2 credentials process.  This is something that gets committed to GitHub?

Reply:
@hpjaj from the research i've done, keifer has a local master key that he would have to share with us via an external method, then that unencrypts this which is actually just a yaml file.